doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -5,28 +5,65 @@
 
 ## Table of Contents
 * [3.1 Introduction](#3.1)
-* [3.2 Measurement Criteria](#3.2)
-  * [3.2.1 Storage and IOPS](#3.2.1)
-* [3.3 Measurement Types](#3.3)
-  * [3.3.1 Performance Measurements](#3.3.1)
-  * [3.3.2 Resiliency Measurements](#3.3.2)
-* [3.4 NFVI Test Cases](#3.4)
+* [3.2 Assumptions](#3.2)
+* [3.3 Requirement Type](#3.3)
+* [3.4 Profile Catalog](#3.4)
+* [3.5 Software & Hardware Reference](#3.5)
+* [3.6 Options & Extensions](#3.6)
+* [3.7 Measurement Criteria](#3.7)
+  * [3.7.1 Storage and IOPS](#3.7.1)
+* [3.8 Measurement Types](#3.8)
+  * [3.8.1 Performance Measurements](#3.8.1)
+  * [3.8.2 Resiliency Measurements](#3.8.2)
+* [3.9 NFVI Test Cases](#3.9)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
 > Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
 
 <a name="3.2"></a>
-## 3.2 Measurement Criteria
+## 3.2 Assumptions
+
+Content to be written:
+- Assumptions:  Automatable, Integrated with CICD tool chain
+
+<a name="3.3"></a>
+## 3.3 Requirement Type
+
+Content to be written:
+-  Type of requirement:  Bare metal, API, etc
+
+<a name="3.4"></a>
+## 3.4 Profile Catalog
+
+Content to be written:
+-  Table showing Profile Catalog
+
+<a name="3.5"></a>
+## 3.5 Software & Hardware Reference
+
+Content to be written:
+- Identify SW Reference
+- Identify HW Reference
+
+<a name="3.6"></a>
+## 3.6 Options & Extensions
+
+Content to be written:
+- Options Available / Configured
+- Extensions Available / Configured
+
+<a name="3.7"></a>
+## 3.7 Measurement Criteria
 
 Test validations will be corroborated, and confirmed, with direct comparison between measured results and documented non-functional requirements (NFRs) for applications, hardware and software configuration settings, and host systems.  Throughput, latency, concurrent connections/threads, are all examples of non-functional requirements which specify criteria which can be used to judge the operation of a system, rather than specific behaviours of the application which are defined by functional requirements.
 
 This section attempts to summarize a categorical list of metrics used for test validations.  **For a complete list of metrics, and requirements, please refer to Reference Model**
 
 
-<a name="3.2.1"></a>
-### 3.2.1 Storage and IOPS
+<a name="3.7.1"></a>
+### 3.7.1 Storage and IOPS
```

Comment:
I am wondering why storage is explicitly mentioned here, but not network throughput, latency, etc.?

Reply:
@gkunz - Storage and IOPS are just a sample.  Network throughput/latency, among other considerations needs to be added.  This is a placeholder to start the document and section.